Source/WSTagsField.swift

```diff
@@ -794,6 +794,9 @@ extension WSTagsField: UITextFieldDelegate {
     }
 
     public func textFieldDidEndEditing(_ textField: UITextField) {
+        if !isTextFieldEmpty{
```

Comment:
```suggestion
        if !isTextFieldEmpty {
```

This project doesn't have a linter but it should have one 😄 
Please add a space between the text and the curly brace. Thank you

Reply:
Done. Added it!